internal/common/logmgr/logmgr.go

```diff
@@ -89,3 +89,12 @@ func GetLogLevel() logrus.Level {
 	}
 	return level
 }
+
```

Comment:
Please add the description of this new function for clearing the scanning report.

Reply:
Done!